src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -124,6 +124,11 @@
   {:name schema/Str
    :files [EnvironmentClassesFileEntry]})
 
+(def EnvironmentModulesInfo
+  "Schema for the return payload an environment_classes request."
+  {:name schema/Str
+   :modules schema/Any}) ; TODO: fixme
```

Comment:
if the value can be null, then yea, `schema/maybe` sounds good. if the key can be missing, then `(schema/optional-key "forge_name")` probably?